docs/arxiv/empirical/experimentation.tex

```diff
@@ -1,2 +1,12 @@
 \section{Experimentation}
 \subsection{Experiment 1: Learning Q functions on components of the Actor network.}
+
+\textbf{Motivation:}
+Our first batch of experiments aimed to establish empirically that the Q-functions of each layer, denoted $Q_1...Q_L$ are similar to the Q-function of the overall network, denoted $Q_{\mu}$. To that end, we extended the actor-critic methodology used in Lillicrap et al (2016) as follows. In addition to training $Q^\mu$, to estimate the Q functions of the actor network $\mu$, sub-critic networks $Q^n$ were initialized for each individual layer. We then compare the $Q$ values estimated by the subcritics to the $Q$ estimation provided by the main critic, $Q^\mu$. \\
```

Comment:
> Q-functions of each layer, denoted $Q_1...Q_L$ 
We are changing this to Q functions of each neuron. Hence $Q^n$ denotes the Q function for each neuron $n$.

This is a recent change so @arkwave @philkuz I know you wrote this according to the experiment at the time, but I would love if we could change it back.